### ISSUE TITLE
Unexpected Draft Creation without User Consent

### ISSUE THREAD

**Issue by ioanmo226** [FlowCrypt/flowcrypt-browser#5380]
It seems that drafts are being created without asking the user, which isn't cool.
Here's how I stumbled on this issue, though there might be more cases:

1. Quit & Reopen Gmail
2. Open a PGP message/thread
3. Chill until you see `Enter passphrase to open this message.`
4. Type in that passphrase
5. Hang around a bit(4-7 seconds) and boom, a new draft is created.

https://github.com/FlowCrypt/flowcrypt-browser/assets/77344191/c352e3d3-d350-4dc7-b4c9-1b2e1fbea15b

This could confuse users and maybe even mess up some workflows.

**Comment by ioanmo226** [comment]
@sosnovsky

**Comment by sosnovsky** [comment]
@ioanmo226 does it happen on `master`?
I wasn't able to reproduce - opened PGP thread, entered passphrase, waited for 30 seconds but new draft wasn't created.

**Comment by ioanmo226** [comment]
Yeah, only need to wait 3-7 seconds (after you enter passphrase).
Maybe you need to open thread with multiple recipients?

**Comment by ioanmo226** [comment]
Updated Issue description and attached video

**Comment by ioanmo226** [comment]
Have you ever noticed that a draft being created unexpectedly on your side without your consent?

**Comment by sosnovsky** [comment]
No, I currently work on extension code every day, but didn't notice such issue

**Comment by ioanmo226** [comment]
Hmm... In my side I see new drafts created frequently.

**Comment by sosnovsky** [comment]
@martgil can you please try to reproduce bug from issue description on your side?

**Comment by ioanmo226** [comment]
Hmmm... Account doesn't matter for me.
For my flowcrypt account, this issue also happens.

**Comment by martgil** [comment]
Thanks, Ioan - I am now continuing to check using the e2e.

**Comment by martgil** [comment]
I'm still unable to replicate the issue on my side using the FlowCrypt browser version 8.5.0 + the e2e account. After I close and re-open Gmail to re-invoke the locked encrypted message -> decrypt the message by providing my passphrase -> waiting for at least 7 seconds or so and going to the draft message section, no draft we're being created.

**Comment by ioanmo226** [comment]
Hmm... Seems like it only happens on my side.
Let me have a look at this issue after I finish iOS 1.2.5 milestone

**Comment by ioanmo226** [comment]
First off, sorry to bother you all with this.

So, get this: I kept running into the problem after building from master.
Then, by pure accident, I deleted the extension and added it back. Guess what? Problem's gone! 🎉

I'm thinking maybe some hidden cache was messing with me? 🤔 Can't get it to happen again, so I'm closing this one out.
But let's stay on our toes in case it decides to make a comeback.

Cheers, and sorry for the noise! 🙏
